src/main/scala/de/bripkens/ha/Configuration.scala

```diff
@@ -25,7 +24,7 @@ object Configuration {
 
 @JsonCreator
 case class Configuration(@JsonProperty("endpoints") endpoints: Set[HealthCheckEndpoint],
-                         @JsonProperty("reporters") reporters: Map[String, AbstractReporterConfig],
+                         @JsonProperty("reporters") reporters: Map[String, ReporterConfig],
```

Comment:
@hseeberger I had that in mind. I'm going to address that soon. First I had to get rid of the explicit type references and the untyped Props creation. I think we have reached a state now, where I can start working on that.